test/unit/cssnano-simple/cssnano-preset-simple/with-escaped-selector.ts

```diff
@@ -1,19 +0,0 @@
-import postcss from 'postcss'
-import mod from './plugin'
-
-describe('with escaped selector', () => {
-  test('should not misplace the backslash', async () => {
-    const input = `
-      .foo\\,2 {
-        background: blue;
-      }
-    `
-
-    const res = await postcss([mod]).process(input, {
-      from: 'input.css',
-      to: 'output.css',
-    })
-
-    expect(res.css).toMatchInlineSnapshot(`".foo\\\\,2{background:blue}"`)
```

Comment:
Written as a test but never actually ran since it didn't include the `.test` extension.
It is apparent now that it never ran since it actually matches `".foo\\,2{background:blue}"` once put into a `.test` fil.e